cvat-core/src/frames.js

```diff
@@ -465,18 +465,12 @@
                     && cachedFrames.length < (this._size * 3) / 4) {
                     const maxFrame = cachedFrames ? Math.max(...cachedFrames) : frameNumber;
                     if (maxFrame < this._stopFrame) {
-                        this.makeFillRequest(maxFrame + 1, frameStep);
+                        await this.makeFillRequest(maxFrame + 1, frameStep);
```

Comment:
Can we change this line as discussed today with smth like this?
```
this.makeFillRequest(maxFrame + 1, frameStep).catch(e => {
    if (e !== 'not needed') {
        throw e;
    }
});
```
Otherwise, we have to wait until `proactive` decoding resolve frames.

Reply:
Guess it will not working as you expect. In simple example we still have same error.
![изображение](https://user-images.githubusercontent.com/8168727/77736499-5b849700-701d-11ea-926e-a431e97a1a0f.png)
@bsekachev what do you think about it?

Reply:
@ActiveChooN 

Let's discuss it in skype